cmd/clusterctl/cmd/pivot.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type pivotOptions struct {
+	fromKubeconfig string
+	toKubeconfig   string
+}
+
+var po = &pivotOptions{}
+
+var pivotCmd = &cobra.Command{
+	Use:   "pivot",
+	Short: "Pivot a management cluster into another management cluster",
+	Long: LongDesc(`
+		Pivot the content of a management cluster, provider components and objects, 
```

Comment:
Ohh, might be I have missed this requirement. Is there an issue?
And, should we preserve the possibility to pivot everything or simply get rid of it?

Reply:
I don't think there was a specific issue, but iirc it was discussed as part of the earlier clusteradm discussions.

I think it would greatly simplify the implementation of pivot, but would potentially introduce more manual steps (deploying the capi and provider components to the new management cluster, configuring/reconfiguring secrets, etc). I don't particularly have a preference on which approach we take, was mainly looking for clarification here :)

Reply:
I think the word pivot could be changed to move or transfer. Pivot used to be move all the things from a to b but now it's just moving the resources, so maybe we can remove the pivot language in order to not confuse the concepts/what this is, in theory, should be doing?

Reply:
> now it's just moving the resources

ATM there is no track of the decision to change the pivot behavior, I'm going to open an issue so this can be sorted out, and then eventually rework the command

Reply:
It would probably be easier to start with move/transfer (with the extra manual steps to prep the target management cluster). And we can consider full pivot for later, if we still want it.

Reply:
Ok, it seems there is agreement on moving only objects, so I change Pivot to Move